app/experimenter/nimbus-ui/src/components/ChangeApprovalOperations/index.test.tsx

```diff
@@ -14,17 +14,20 @@ import {
   reviewTimedOutBaseProps,
 } from "./mocks";
 
+const REVIEW_URL =
+  "http://localhost:8888/v1/admin/#/buckets/main-workspace/collections/nimbus-mobile-experiments/records";
+
 const Subject = ({
   rejectChange = () => {},
   approveChange = () => {},
-  startRemoteSettingsApproval = () => {},
+  reviewUrl = REVIEW_URL,
```

Comment:
I don't see any tests modifying this prop, so it could probably be removed and REVIEW_URL passed directly into the component.